run.py

```diff
@@ -0,0 +1,4 @@
+from app import app
+
+if __name__ == "__main__":
+    app.run(debug=True)
```

Comment:
You still have debug=True always ;) please remove it and use env. I am not sure if you need directly use env or just Flask by defáult use env variables.

Reply:
Although I highly appreciate your commitment to mentoring me (<3) please enjoy your well deserved vacation!!! My debug (which is now fixed btw) can wait a couple of days!!